config.go

```diff
@@ -18,11 +18,6 @@ type Config struct {
 	// a leader before we attempt an election.
 	ElectionTimeout time.Duration
 
-	// CommitTimeout controls the time without an Apply() operation
```

Comment:
The plan is to accelerate heartbeats. We don't think that'll be too expensive, and we'll find out through benchmarking if we're wrong.